### PR TITLE
port alert example to 0.5

### PR DESCRIPTION
**Pull request by euclio**
#### Summary

Port the `alert` example to 0.5 syntax. cc #100.

#### Checklist

- [x] cargo fmt
- [x] cargo clippy
- [x] cargo test
- [x] updated CHANGES.md

### CODE REVIEW

File: examples/Cargo.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dev-dependencies]
 relm4 = { path = "../relm4", features = ["macros"] }
+relm4-components = { path = "../relm4-components" }
```
Comment (AaronErhardt) [inline examples/Cargo.toml:9]: I think we should rather place it here, right? https://github.com/Relm4/Relm4/tree/main/examples/relm4-components

**Review by AaronErhardt — CHANGES_REQUESTED** [review]
LGTM. It just makes sense to use the relm4-components folder for related examples since we already have it.

File: examples/Cargo.toml (hunk 1 above)
Comment (euclio) [reply examples/Cargo.toml]: Yes, fixed.

**Review by AaronErhardt — APPROVED** [review]
Thanks!

**Pull request merged**